propagator/xray/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Release History: opentelemetry-propagator-xray
+### v0.18.0 / 2021-04-28
+* Fix bug on parsing trace id format
```

Comment:
Thanks for this, however the release tooling will populate this automatically (and will complain that it is already populated).
```suggestion
```

Reply:
unpopulated